src/platform/testing/e2e/cypress/support/commands/axeCheck.js

```diff
@@ -28,12 +28,20 @@ const processAxeCheckResults = violations => {
  * Checks the current page for aXe violations.
  * @param {string} [context] - Selector for the container element to aXe check.
  */
-Cypress.Commands.add('axeCheck', (context = 'main') => {
+Cypress.Commands.add('axeCheck', (context = 'main', tempOptions = {}) => {
+  const { _13647Exception } = tempOptions;
   Cypress.log();
 
-  const options = {
-    includedImpacts: ['critical'],
-  };
+  const options = _13647Exception
+    ? {
+        includedImpacts: ['critical'],
+      }
+    : {
+        runOnly: {
+          type: 'tag',
+          values: ['wcag2a', 'wcag2aa'],
+        },
+      };
```

Comment:
Running axe checks using the `wcag2a` and `wcag2aa` rulesets cause some existing cypress checks to break for a few apps. This is set up so that we can have the stricter checks in place immediately, but still allow the previous level of axe checking on the few apps which fail under the stricter checks.